### PR TITLE
Larger mint buttons, fix Mint Invite Bonuses header

### PR DESCRIPTION
**Pull request by markulrich**
Omar: Updated to make it look better.

### Screenshots

img_7862

img_7863

### CODE REVIEW

**Comment by markulrich** [comment]
simulator screen shot - iphone 6 - 2018-10-09 at 00 23 16

simulator screen shot - iphone 6 - 2018-10-09 at 00 23 04

File: src/components/shared/MintButton.tsx [hunk 1]
```diff
@@ -74,15 +75,21 @@ const MintButtonComponent: React.StatelessComponent<Props> = props => {
...
               role: CurrencyRole.None
             },
-            "per week."
+            "a week, limit",
+            {
+              currencyType: CurrencyType.Raha,
+              value: new Big(RAHA_MINT_MAX),
+              role: CurrencyRole.None
+            },
+            "per MINT."
```
Comment (osdiab) [inline src/components/shared/MintButton.tsx]: all caps mint is weird. let's make it

"You may create/mint R10.00 per week up to a maximum of R40.00 at once."

Or maybe,

Basic Income Rate: R10.00 per week
Max outstanding basic income: R40.00

File: src/components/shared/MintButton.tsx [hunk 2]
```diff
@@ -74,15 +75,21 @@ const MintButtonComponent: React.StatelessComponent<Props> = props => {
       />
       <Text style={{ marginTop: 4 }}>
         <MixedText
-          style={[fontSizes.small]}
+          style={[fontSizes.small, { textAlign: "center" }]}
           content={[
-            "The current mint rate is",
+            "Create ",
```
Comment (osdiab) [inline src/components/shared/MintButton.tsx]: Do we want to use the word "Create" or "Mint"? We keep switching between the two, i think we should choose one and commit. Mint is more conceptually compelling but Create is clearer.

File: src/components/pages/Wallet.tsx [hunk 3]
```diff
@@ -140,16 +141,14 @@ const Invite: React.StatelessComponent<Props> = props => {
...
   return (
     <React.Fragment>
-      <Text style={[styles.inviteSectionText]}>
-        You have nothing to mint at this time.
-      </Text>
+      <Text style={[styles.inviteSectionText]}>Nothing to mint right now.</Text>
       <View>
         <Text style={styles.inviteSectionText}>
-          Invite a friend to earn 60 Raha:
+          Invite a friend to create {REFERRAL_BONUS} Raha!
```
Comment (osdiab) [inline src/components/pages/Wallet.tsx]: Same mention here, inconsistent wording: Mint or Create?

File: src/components/pages/Wallet.tsx [hunk 4]
```diff
@@ -140,16 +141,14 @@ const Invite: React.StatelessComponent<Props> = props => {
...
         </Text>
         <Button
           style={styles.button}
-          title="Invite"
+          title={`INVITE +${REFERRAL_BONUS}`}
```
Comment (osdiab) [inline src/components/pages/Wallet.tsx]: You don't need to capitalize it, use normal capitalization: Invite +${REFERRAL_BONUS}

File: src/components/pages/Wallet.tsx [hunk 5]
```diff
@@ -187,7 +186,9 @@ const inviteSectionTextStyle: TextStyle = {
 };
 
 const buttonStyle: ViewStyle = {
-  marginVertical: 8
+  marginVertical: 8,
+  paddingVertical: 16,
+  width: 300
```
Comment (tinaroh) [inline src/components/pages/Wallet.tsx]: minWidth? This will probably break depending on font size / i18n

Comment (osdiab) [reply src/components/pages/Wallet.tsx]: why does it need an explicit width? can't it just be made consistently good with proper padding? What motivated this change?

Comment (osdiab) [reply src/components/pages/Wallet.tsx]: if you want it to be full width you can probably just go with `width: "100%"` or something using `flex` sizing to grow to full width, depending on how this page's styles are structured. One-off random numbers like this is how we end up with giant styling headaches.

File: src/components/pages/Wallet.tsx [hunk 6]
```diff
@@ -140,18 +141,16 @@ const Invite: React.StatelessComponent<Props> = props => {
...
+      <Text style={[styles.inviteSectionText]}>Nothing to mint right now.</Text>
       <View>
         <Text style={styles.inviteSectionText}>
-          Invite a friend to earn 60 Raha:
+          Invite a friend to mint {REFERRAL_BONUS} Raha!
         </Text>
         <Button
           style={styles.button}
-          title="Invite"
+          title={`Invite +${REFERRAL_BONUS}`}
```
Comment (osdiab) [inline src/components/pages/Wallet.tsx]: We can actually make it have the full R 60.00, like how `MintButton` does it; you can do something like `title={["Invite +", referralBonus]}` where `referralBonus` is the actual Currency (with `CurrencyRole.None` to use the same color) object referring to that referral bonus. That makes it clearer it's an amount of currency and is also more consistent

File: src/components/pages/Wallet.tsx [hunk 7]
```diff
@@ -187,7 +186,9 @@ const inviteSectionTextStyle: TextStyle = {
 };
 
 const buttonStyle: ViewStyle = {
-  marginVertical: 8
+  marginVertical: 8,
+  paddingVertical: 16,
+  minWidth: 300
```
Comment (osdiab) [inline src/components/pages/Wallet.tsx]: You can push with this, but why still `300`? If you want it to be full width I think there are better ways than this... I think `width: 100%` should work, but if it doesn't you can put the button in a `View` with `flexDirection: row` and giving the button `flex: 1`. The parent view would automatically fill to full width and the button would then grow to full width of the parent. `300` might look good on your device but isn't flexible to others and makes for inconsistent visuals in the app.

Comment (osdiab) [reply src/components/pages/Wallet.tsx]: If you just don't want to deal with this right now, then I'll make the button component more expressive to our needs later on. The fact you'd like to change the padding for this button and this one only seems to reflect that the button styles are not generalizing to different situations, so having choices for standardized button sizing is probably useful as well.

Comment (osdiab) [reply src/components/pages/Wallet.tsx]: although from the screenshots i cant actually tell what changing the button padding is doing. can you just remove that line?

**Comment by markulrich** [comment]
simulator screen shot - iphone 6 - 2018-10-10 at 00 06 38

File: src/components/shared/Navigation.tsx [hunk 8]
```diff
@@ -372,7 +372,7 @@ const WalletTab = createNavigatorForTab(
     },
     [RouteName.ReferralBonusPage]: {
       screen: ReferralBonus,
-      navigationOptions: createHeaderNavigationOptions()
+      navigationOptions: createHeaderNavigationOptions("Mint referral bonus", true)
```
Comment (tinaroh) [inline src/components/shared/Navigation.tsx]: Can we just call it Referrals? Seems like a long header that could get chopped off on smaller phones

Comment (osdiab) [reply src/components/shared/Navigation.tsx]: done

File: src/components/pages/Wallet.tsx (hunk 7 above)
Comment (osdiab) [reply src/components/pages/Wallet.tsx]: fixed this.

**Comment by osdiab** [comment]
@tinaroh please take a look, i made changes, see what it looks like from screenshots on the description of the issue.

**Pull request merged**